### PR TITLE
Pool fees

### PR DESCRIPTION
**Pull request by sophialittlejohn**
### Description

This pull request...

- adds hardcoded pool fee to create pool form
- adds Fees tab
- adds ability to add and remove fees (adding fees through fees tab will require the admin and for the changes to be applied before they are charged)
- adds ability to set a value to charge on existing fees - charging can only be done by the address which will receive the fees

#1859 

Closes #<issue-number>

### Approvals

- [ ] Dev
- [ ] Dev
- [ ] Designer
- [ ] Product

### Screenshots

Screenshot 2024-01-29 at 17 50 27

----

Screenshot 2024-01-29 at 17 50 38

----

Screenshot 2024-01-29 at 17 50 47

### CODE REVIEW

File: centrifuge-app/src/components/PoolFees/ChargeFeesDrawer.tsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            <Text variant="label2">Receiving address</Text>
+            <CopyToClipboard variant="body3" address={feeChainData?.destination || ''} />
+          </Stack>
+        </Shelf>
+        <Stack bg="backgroundTertiary" p={2}>
+          {feeChainData?.amounts.pending.gtn(0) && !updateCharge ? (
+            <Stack gap={2}>
+              <Stack gap={1} bg="backgroundButtonSecondary" p={1} borderRadius="2px">
+                <Shelf gap={1} alignItems="baseline">
+                  <Text variant="body3" color="gray.800">
```
Comment (onnovisser) [inline centrifuge-app/src/components/PoolFees/ChargeFeesDrawer.tsx:104]: `gray.800` can be replaced with `textSecondary`

File: centrifuge-app/src/components/PoolFees/ChargeFeesDrawer.tsx [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  return (
+    <Drawer isOpen={isOpen} onClose={onClose}>
+      <Stack gap={3}>
+        <Text textAlign="center" variant="heading2">
+          Charge {feeMetadata?.name}
+        </Text>
+        <Shelf
+          borderTop="0.5px solid"
+          borderBottom="0.5px solid"
+          borderColor="#DDDBD9"
```
Comment (onnovisser) [inline centrifuge-app/src/components/PoolFees/ChargeFeesDrawer.tsx:72]: can just use `borderPrimary` which is basically the same color as this kinda random one

**Review by onnovisser — APPROVED** [review]
💯

**Comment by TimHoub** [comment]
I changed the design a little bit. I didn't really like it when I saw it implemented and made one tiny change.
The background of the page should be all white instead of #F5F5F5. Hope this is not a big hassle, but it just cleans up the page a lot! https://www.figma.com/file/cn5CiQsuVDOmieszqcneX2/Pool-management?type=design&node-id=512%3A2723&mode=design&t=WRMHgB7iEhOUiSDq-1

**Pull request merged**